modules/terraform/plan.go

```diff
@@ -41,6 +41,28 @@ func PlanE(t testing.TestingT, options *Options) (string, error) {
 	return RunTerraformCommandE(t, options, FormatArgs(options, "plan", "-input=false", "-lock=false")...)
 }
 
+// InitAndPlanAndShow runs terraform init, then terraform plan, and then terraform show with the given options, and
+// returns the json output of the plan file. This will fail the test if there is an error in the command.
+func InitAndPlanAndShow(t testing.TestingT, options *Options) string {
```

Comment:
Thinking out loud, what's the difference between:

1. `terraform show -json`
1. `terraform plan -out=x && terraform show -json x`

I guess (1) shows the contents of the state file in JSON format, while (2) is a way to see the contents of a `plan` in JSON format? And I guess the only reason we need (2) is because `plan` doesn't natively support a `-json` flag?

Reply:
Yes that's right. E.g., if you do `terraform show -json` on an unapplied module, you won't see anything.

Reply:
Roger